### PR TITLE
Doc lib update

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 6517661 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-549.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by cbordier** [comment]
@lbadenhop ready to merge.

File: content/en/docs/manage_document_library/documentation_library_management.md [hunk 1]
```diff
@@ -30,8 +30,11 @@ View the following information for all documents:
 * Description - the document description (may be truncated after 50 characters)
 * Access - the document visibility (Public / Protected)
 * Referenced - the number of times the document is referenced (product document / Marketplace settings)
-* Type - the document type (plain text format - TXT / Portable Document Format - PDF / Word documents - DOC, DOCX / Markdown - MD / PowerPoint - PPTX / Excel - XLSX)
+* Type - the document type:
+    * Files format: Portable Document Format - PDF / Word documents - DOC, DOCX / Markdown - MD / PowerPoint - PPTX / Excel - XLSX
```
Comment (lbadenhop) [inline content/en/docs/manage_document_library/documentation_library_management.md:34]:
```suggestion
    * Files format: Portable Document Format - PDF / Word documents - DOC, DOCX / Markdown - MD / PowerPoint - PPTX / Excel - XLSX / TXT
```

File: content/en/docs/manage_document_library/documentation_library_management.md [hunk 2]
```diff
@@ -44,14 +47,17 @@ To view a document's details:
 
     * Document name - the display name for the product in the WebUI
     * Document logical name - the internal name used to access the document
-    * Document type - TXT / PDF / DOC / DOCX / MD / PPTX / XLSX
+    * Document type - TXT / PDF / DOC / DOCX / MD / PPTX / XLSX / URL
```
Comment (lbadenhop) [inline content/en/docs/manage_document_library/documentation_library_management.md:50]: URL is a document type?

File: content/en/docs/manage_document_library/documentation_library_management.md [hunk 3]
```diff
@@ -68,37 +74,49 @@ To download a document:
 2. Click on a document name in the list view to open the Document Details.
 3. Click the download button located below the thumbnail to download the document locally.
 
-## Create a document from existing files
+## Create a document
+
+Document library supports multiple document types:
 
-Only PDF, MD, DOC, DOCX, XLSX and PPTX files are accepted. All files larger than 5MB will be rejected.
+* Existing file (PDF, MD, DOC, DOCX, XLSX and PPTX no larger than 5MB) - these files are coming from an external source and cannot be modified from the Document Library. The use will only have the possibility to download them. Only PDF files will show a preview of its content,
```
Comment (lbadenhop) [inline content/en/docs/manage_document_library/documentation_library_management.md:81]:
```suggestion
* Existing file (TXT, PDF, MD, DOC, DOCX, XLSX and PPTX no larger than 5MB) - these files are coming from an external source and cannot be modified from the Document Library. The use will only have the possibility to download them. Only PDF files will show a preview of its content,
```

**Comment by lbadenhop** [comment]
@cbordier, see comments

File: content/en/docs/manage_document_library/documentation_library_management.md (hunk 1 above)
Comment (cbordier) [reply content/en/docs/manage_document_library/documentation_library_management.md:34]: There is no TXT file format.

File: content/en/docs/manage_document_library/documentation_library_management.md (hunk 2 above)
Comment (cbordier) [reply content/en/docs/manage_document_library/documentation_library_management.md:50]: Yes it is consider as Document type for the filtering.

**Pull request merged**